### PR TITLE
Feature/weaviate unit test

### PR DESCRIPTION
**Pull request by hotbrainy**
Added vectordb/webiate unittest code.

### CODE REVIEW

File: libs/tests/schema2.sql
Comment (kirill-andr) [inline libs/tests/schema2.sql:1]: @hotbrainy do we need this file if it is empty?

File: libs/tests/public.sql
Comment (kirill-andr) [inline libs/tests/public.sql:1]: @hotbrainy do we need this file if it is empty?

File: .idea/.gitignore
Comment (kirill-andr) [inline .idea/.gitignore:1]: @hotbrainy the directory .idea should be in .gitignore. It should not be part of the PR

File: .idea/analitiq.iml
Comment (kirill-andr) [inline .idea/analitiq.iml:1]: @hotbrainy the directory .idea should be in .gitignore. It should not be part of the PR

File: .idea/misc.xml
Comment (kirill-andr) [inline .idea/misc.xml:1]: @hotbrainy the directory .idea should be in .gitignore. It should not be part of the PR

File: poetry.lock.backup
Comment (kirill-andr) [inline poetry.lock.backup:1]: @hotbrainy  does this file need to be part of the PR?

**Review by kirill-andr — CHANGES_REQUESTED** [review]

File: libs/tests/schema2.sql
Comment (hotbrainy) [reply libs/tests/schema2.sql:1]: if we dont need, we can remove both in file and code.

File: .idea/.gitignore
Comment (hotbrainy) [reply .idea/.gitignore:1]: OK, will fix that

File: .idea/analitiq.iml
Comment (hotbrainy) [reply .idea/analitiq.iml:1]: OK, will fix that

File: .idea/misc.xml
Comment (hotbrainy) [reply .idea/misc.xml:1]: OK, will fix that

File: .gitignore
Comment (hotbrainy) [inline .gitignore:1]: I added `.idea ` folder here

**Comment by kirill-andr** [comment]
Closing it as the test was manually merged into Master

**Pull request closed without merging**